lite2/errors.go

```diff
@@ -38,3 +38,9 @@ type ErrInvalidHeader struct {
 func (e ErrInvalidHeader) Error() string {
 	return fmt.Sprintf("invalid header: %v", e.Reason)
 }
+
+type ErrNoWitnesses struct{}
```

Comment:
godoc please

Reply:
also, what is the reason behind exposing this error?

Reply:
My idea was that I wanted to standardize some of the errors. I only chose to have it exposed because the rest of the errors were also exposed

Reply:
> My idea was that I wanted to standardize some of the errors.

Can this be done in a separate PR? It's not a good idea to mix a lot of issues in one PR

Reply:
https://github.com/tendermint/tendermint/issues/4538